dataland-frontend/src/components/forms/parts/kpiSelection/EuTaxonomyKPIsModel.ts

```diff
@@ -24,35 +24,39 @@ export const euTaxonomyKPIsModel = {
 };
```

Comment:
Currently these tooltips here are only used on the upload pages. On the display pages they are hardcoded. We should keep this in mind when we refactor the framework at a later point in time.